python_modules/dagster-graphql/dagster_graphql/schema/external.py

```diff
@@ -334,6 +336,14 @@ def resolve_assetGroups(self, _graphene_info: ResolveInfo):
             for group_name, external_nodes in groups.items()
         ]
 
+    def resolve_topLevelResources(self, _graphene_info):
```

Comment:
topLevelResourceDetails?

Reply:
👍 

going to move the plural to `allTopLevelResourceDetails` since pluralizing `...Details` is a mess